test/runtests.sh

```diff
@@ -7,7 +7,7 @@ then
 fi
 
 # delete output from previous run
-if [ ! -d "../actual.txt" ]
+if [ -e "./actual.txt" ]
```

Comment:
I think the `../` is correct since this script is meant to be run in the `test` directory?